Examples/MagicWeatherSwiftUI/Shared/Sources/Lifecycle/PurchasesDelegateHandler.swift

```diff
@@ -18,7 +18,10 @@ class PurchasesDelegateHandler: NSObject, ObservableObject {
 
 extension PurchasesDelegateHandler: PurchasesDelegate {
     
-    /// -  Whenever the `shared` instance of Purchases updates the PurchaserInfo cache, this method will be called.
+    /* Whenever the `shared` instance of Purchases updates the PurchaserInfo cache, this method will be called.
+    
+     Note: PurchaserInfo is not pushed to each Purchases client, it has to be fetched. This delegate method is only called when the SDK updates its cache after an app launch, purchase, restore, or fetch. You still need to call `Purchases.shared.purchaserInfo` to fetch PurchaserInfo regularly.
```

Comment:
maybe split this comment into multiple lines so horizontal scrolling isn't required (xcode auto-translates into multiple lines but some other IDEs don't)